Assets/Scripts/Photos/PhotoAlbum.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+class PhotoAlbum : MonoBehaviour
+{
+    public RawImage albumPhoto, albumPhoto1, albumPhoto2, albumPhoto3;
+    public Button exitAlbumButton, nextPageButton, backPageButton;
+    public PhotoCamera photoCamera;
+    private int currentPage;
+    private int pageSize;
+
+    public void Start()
+    {
+        currentPage = 0;
+        pageSize = 4;
+
+        exitAlbumButton.onClick.AddListener(ExitAlbumOnClick);
+        nextPageButton.onClick.AddListener(NextPageOnClick);
+        backPageButton.onClick.AddListener(BackPageOnClick);
+        backPageButton.gameObject.SetActive(false);
+
+        LoadAlbumPage();
+    }
+
+    private void LoadAlbumPage(int skip = 0)
+    {
+        List<PhotoDTO> photos = PhotoCollectionDTO.PagePhotoCollection(pageSize, skip).ToList();
+
+        if(photos.ElementAt(0) != null)
+        {
+            Texture2D tex = new Texture2D((int)albumPhoto.rectTransform.rect.width, (int)albumPhoto.rectTransform.rect.height);
+            tex.LoadImage(photos.ElementAt(0).PhotoData);
+            albumPhoto.texture = tex;
+        }
+
+        if(photos.ElementAt(1) != null)
```

Comment:
currently throwing an index exception in scene

Reply:
Fixed!